src/main/java/pl/oskarkowalski/recruitment/ticketbookingapp/enums/TicketPrice.java

```diff
@@ -0,0 +1,31 @@
+package pl.oskarkowalski.recruitment.ticketbookingapp.enums;
+
+import lombok.Getter;
+
+@Getter
+public enum TicketPrice {
+  ADULT((double) 25),
```

Comment:
dlaczego trzymanie $$ w double to zły pomysł?

Reply:
Faktycznie, pisząc powyższy kod byłem skupiony na tym, aby po prostu to nie była liczba całkowita. Double jest słaby do pieniędzy, ponieważ jest niewystarczająco dokładny, gdyż jest reprezentowany przez potęgi dwójki. Może to rodzić problemy z dokładnością. Do sum pieniężnych (jak również i powyżej) dobrze jest zastosować BigDecimala.